### ISSUE TITLE
Ansible 2.0 bug

### ISSUE THREAD

**Issue by Lagrang** [bertvv/ansible-role-bind#15]
Try run playbook with your role using Ansible 2.0. It ends with error:

```
ERROR! 'version' is not a valid attribute for a RoleMetadata

The error appears to have been in '/etc/ansible/roles/bertvv.bind/meta/main.yml': line 2, column 1, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
galaxy_info:
^ here
```

**Comment by bertvv** [comment]
`version:` is an unofficial field that is used by a third-party tool for managing role dependencies (`librarian-ansible`). Custom meta fields are no longer accepted in Ansible 2.0. See [ansible/ansible#13496](https://github.com/ansible/ansible/issues/13496) for more info.

I will remove `version:` from the next release of this role.

**Comment by wyaeld** [comment]
@bertvv Are you planning to release this fix soon?

**Comment by bertvv** [comment]
Fixed in v3.3.1. Sorry for the wait!
